### ISSUE TITLE
accuracy 

### ISSUE THREAD

**Issue by ZhenboZhao77** [taugeren/PointOBB-v2#7]
Hello Author!

I apologize for the intrusion. I wanted to ask about using the point2rbox-v2 model. Following your suggestion to reduce the learning rate on a single GPU, I still haven't achieved accuracy comparable to what is reported in the paper on the dota-v1.0 dataset. Below are my evaluation results:

This is your evaluation result for task 1 (VOC metrics):

mAP: 0.4107441461920254
ap of each class: plane: 0.6653970189150719, baseball-diamond: 0.47677691270253497, bridge: 0.059736286297988446, ground-track-field: 0.314252522476226, small-vehicle: 0.42343924145515544, large-vehicle: 0.3567267259625493, ship: 0.4588567486641143, tennis-court: 0.8579662876150658, basketball-court: 0.5423920361843146, storage-tank: 0.511727206303712, soccer-ball-field: 0.14498219445004368, roundabout: 0.49424490169228236, harbor: 0.08461848455361268, swimming-pool: 0.48158188493331666, helicopter: 0.28846374067439173.

**Comment by taugeren** [comment]
Could you please provide the training config and some logs, thank you.

**Comment by g973839433** [comment]
> Hello Author!
> 
> I apologize for the intrusion. I wanted to ask about using the point2rbox-v2 model. Following your suggestion to reduce the learning rate on a single GPU, I still haven't achieved accuracy comparable to what is reported in the paper on the dota-v1.0 dataset. Below are my evaluation results:
> 
> This is your evaluation result for task 1 (VOC metrics):
> 
> mAP: 0.4107441461920254 ap of each class: plane: 0.6653970189150719, baseball-diamond: 0.47677691270253497, bridge: 0.059736286297988446, ground-track-field: 0.314252522476226, small-vehicle: 0.42343924145515544, large-vehicle: 0.3567267259625493, ship: 0.4588567486641143, tennis-court: 0.8579662876150658, basketball-court: 0.5423920361843146, storage-tank: 0.511727206303712, soccer-ball-field: 0.14498219445004368, roundabout: 0.49424490169228236, harbor: 0.08461848455361268, swimming-pool: 0.48158188493331666, helicopter: 0.28846374067439173.

你好，你现在的map可以实现原文的结果了么

**Comment by ZhenboZhao77** [comment]
很遗憾，并没有。我达不到论文的精度。请问你能达到吗？


---- Replied Message ----
| From | ***@***.***> |
| Date | 11/4/2024 18:42 |
| To | ***@***.***> |
| Cc | ***@***.***>,
***@***.***> |
| Subject | Re: [taugeren/PointOBB-v2] accuracy (Issue #7) |

Hello Author!

I apologize for the intrusion. I wanted to ask about using the point2rbox-v2 model. Following your suggestion to reduce the learning rate on a single GPU, I still haven't achieved accuracy comparable to what is reported in the paper on the dota-v1.0 dataset. Below are my evaluation results:

This is your evaluation result for task 1 (VOC metrics):

mAP: 0.4107441461920254 ap of each class: plane: 0.6653970189150719, baseball-diamond: 0.47677691270253497, bridge: 0.059736286297988446, ground-track-field: 0.314252522476226, small-vehicle: 0.42343924145515544, large-vehicle: 0.3567267259625493, ship: 0.4588567486641143, tennis-court: 0.8579662876150658, basketball-court: 0.5423920361843146, storage-tank: 0.511727206303712, soccer-ball-field: 0.14498219445004368, roundabout: 0.49424490169228236, harbor: 0.08461848455361268, swimming-pool: 0.48158188493331666, helicopter: 0.28846374067439173.

你好，你现在的map可以实现原文的结果了么

**Comment by taugeren** [comment]
请问可以发一下log文件嘛，我可以来看看是否有什么配置不一样的地方

**Comment by ZehaoGao2000** [comment]
> 很遗憾，并没有。我达不到论文的精度。请问你能达到吗？
> […](#)
> ---- Replied Message ---- | From | ***@***.***> | | Date | 11/4/2024 18:42 | | To | ***@***.***> | | Cc | ***@***.***>, ***@***.***> | | Subject | Re: [taugeren/PointOBB-v2] accuracy (Issue #7) | Hello Author! I apologize for the intrusion. I wanted to ask about using the point2rbox-v2 model. Following your suggestion to reduce the learning rate on a single GPU, I still haven't achieved accuracy comparable to what is reported in the paper on the dota-v1.0 dataset. Below are my evaluation results: This is your evaluation result for task 1 (VOC metrics): mAP: 0.4107441461920254 ap of each class: plane: 0.6653970189150719, baseball-diamond: 0.47677691270253497, bridge: 0.059736286297988446, ground-track-field: 0.314252522476226, small-vehicle: 0.42343924145515544, large-vehicle: 0.3567267259625493, ship: 0.4588567486641143, tennis-court: 0.8579662876150658, basketball-court: 0.5423920361843146, storage-tank: 0.511727206303712, soccer-ball-field: 0.14498219445004368, roundabout: 0.49424490169228236, harbor: 0.08461848455361268, swimming-pool: 0.48158188493331666, helicopter: 0.28846374067439173. 你好，你现在的map可以实现原文的结果了么 — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you authored the thread.Message ID: ***@***.***>

![微信截图_20241108195753](https://github.com/user-attachments/assets/0114b50c-1ec6-4a16-bf7c-17f464934795)

我复现到了与论文相似的精度。

**Comment by ZhenboZhao77** [comment]
ok！那应该是我的问题，可能是忘记之前更改过mmdet下配置了。


--- Replied Message ----
| From | ***@***.***> |
| Date | 11/8/2024 19:59 |
| To | ***@***.***> |
| Cc | ***@***.***>,
***@***.***> |
| Subject | Re: [taugeren/PointOBB-v2] accuracy (Issue #7) |

很遗憾，并没有。我达不到论文的精度。请问你能达到吗？
…
---- Replied Message ---- | From | @.> | | Date | 11/4/2024 18:42 | | To | @.> | | Cc | @.>, @.> | | Subject | Re: [taugeren/PointOBB-v2] accuracy (Issue #7) | Hello Author! I apologize for the intrusion. I wanted to ask about using the point2rbox-v2 model. Following your suggestion to reduce the learning rate on a single GPU, I still haven't achieved accuracy comparable to what is reported in the paper on the dota-v1.0 dataset. Below are my evaluation results: This is your evaluation result for task 1 (VOC metrics): mAP: 0.4107441461920254 ap of each class: plane: 0.6653970189150719, baseball-diamond: 0.47677691270253497, bridge: 0.059736286297988446, ground-track-field: 0.314252522476226, small-vehicle: 0.42343924145515544, large-vehicle: 0.3567267259625493, ship: 0.4588567486641143, tennis-court: 0.8579662876150658, basketball-court: 0.5423920361843146, storage-tank: 0.511727206303712, soccer-ball-field: 0.14498219445004368, roundabout: 0.49424490169228236, harbor: 0.08461848455361268, swimming-pool: 0.48158188493331666, helicopter: 0.28846374067439173. 你好，你现在的map可以实现原文的结果了么 — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you authored the thread.Message ID: @.***>

_20241108195753.png (view on web)

我复现到了与论文相似的精度。
